library/functions.lisp

```diff
@@ -122,6 +122,7 @@
 
   (declare /= (Eq :a => :a -> :a -> Boolean))
   (define (/= a b)
+    "Returns TRUE if a is not equal to b."
```

Comment:
I like a more informal way of writing: "Is A not equal to B?"

Reply:
(Regardless of how you decide, it should be written `\`a\`` or `A`; and `True` not `TRUE` as is conventional for ADT constructors.)